cfme/tests/configure/test_zones.py

```diff
@@ -65,11 +64,14 @@ def test_zone_change_appliance_zone(request):
     @request.addfinalizer
     def _return_zone_back():
         current_appliance.server.zone = current_appliance.default_zone
-    request.addfinalizer(conf.BasicInformation(appliance_zone="default").update)
-    basic_info = conf.BasicInformation(appliance_zone=zone.name)
-    basic_info.update()
-    current_appliance.server.zone = zone
-    assert zone.description == store.current_appliance.zone_description
+
+    server_settings = appliance.server.settings
+    request.addfinalizer(lambda: server_settings.update_basic_information(
+        {'appliance_zone': "default"}))
+    server_settings.appliance.server.zone = zone
```

Comment:
**QUESTION:** Why not just ```appliance.server.zone```?

Reply:
I should be appliance.server.zone, I missed it